Core/GDCore/IDE/Dialogs/LayoutEditorCanvas/LayoutEditorCanvasOptions.h

```diff
@@ -49,6 +49,8 @@ class GD_CORE_API LayoutEditorCanvasOptions {
   int gridB;         ///< Grid blue color in editor
   float zoomFactor;  ///< Stores the zoom factor
   bool windowMask;   ///< True if window mask displayed in editor
+  int xValue;
+  int yValue;

```

Comment:
"value" is implied by the fact that... it's a variable, so it's a value of something :)
"x" and "y" is too vague, it's the coordinates of what?
=> prefer `viewXPosition` and `viewYPosition` (and `viewPositionValid` as discussed in another comment)

Reply:
(I agree it's a bit of nitpicking, but naming variables properly is important to keep things clear and precise as this can grow with other variable members in the future ;))

Reply:
modified